webpack/assets/javascripts/react_app/components/MemoryAllocationInput/__tests__/__snapshots__/MemoryAllocationInput.test.js.snap

```diff
@@ -0,0 +1,148 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`MemoryAllocationInput rendering should render with default props 1`] = `
+<Grid
+  bsClass="container"
+  componentClass="div"
+  fluid={false}
+>
+  <Row
+    bsClass="row"
+    componentClass="div"
+  >
+    <Col
+      bsClass="col"
+      componentClass="div"
+    >
+      <NumericInput
+        className=""
+        disabled={false}
+        format={[Function]}
+        label="Memory"
+        minValue={0}
+        name=""
+        onChange={[Function]}
+        parser={[Function]}
+        precision={0}
+        readOnly={false}
+        step={1}
+        value={1}
+      />
+    </Col>
+  </Row>
+  <Row
+    bsClass="row"
+    componentClass="div"
+  >
+    <Col
+      bsClass="col"
+      componentClass="div"
+      md={2}
+    />
+    <Col
+      bsClass="col"
+      className="form-group"
+      componentClass="div"
+      md={4}
+    />
+  </Row>
+</Grid>
+`;
+
+exports[`MemoryAllocationInput rendering should render with error message 1`] = `
```

Comment:
Looks like this snap is the same as the default props snap. This one should have an error message

Reply:
I checked the error and warning states in a different way. Let me know if it looks ok now.